libcodechecker/version.py

```diff
@@ -23,3 +23,10 @@
 CLIENT_API = '{0}.{1}'.format(
     max(SUPPORTED_VERSIONS.keys()),
     SUPPORTED_VERSIONS[max(SUPPORTED_VERSIONS.keys())])
+
+
+def get_version_str():
+    versions = ""
+    for k, v in SUPPORTED_VERSIONS.items():
```

Comment:
If multiple supported versions are available, it will write the following: `v6.7v7.1`.
Maybe this code can be written like so:
```py
', '.join(["v" + str(v) + "." + str(SUPPORTED_VERSIONS[v]) for v in SUPPORTED_VERSIONS])
```
This way it will write the following: `v6.7, v7.1`

Reply:
done